src/sentry/sentry_metrics/multiprocess.py

```diff
@@ -332,6 +339,29 @@ def join(self, timeout: Optional[float] = None) -> None:
         self.__producer.close()
 
 
+def valid_metric_name(name: str) -> bool:
+    if not name:
+        return False
+    if len(name) > MAX_NAME_LENGTH:
+        return False
+
+    return True
+
+
+def valid_metric_tags(tags: Mapping[str, str]) -> Tuple[bool, Sequence[str]]:
+    invalid_strs: List[str] = []
+    is_valid = True
+    for key, value in tags.items():
+        if not key or len(key) > MAX_TAG_KEY_LENGTH:
+            invalid_strs.append(key)
+            is_valid = False
+        if not value or len(value) > MAX_TAG_VALUE_LENGTH:
+            invalid_strs.append(value)
+            is_valid = False
+
+    return (is_valid, invalid_strs)
```

Comment:
I don't feel strongly but `is_valid` is always going to be equal to `len(invalid_strs) == 0` if we want to drop that from the signature